frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -576,7 +576,7 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
             isChecked={enableMonitoring}
             data-checked-state={enableMonitoring}
           />
-          {props.packageManifest.data[0].metadata.labels['opsrc-provider'] !== 'redhat' && (
+          {props.packageManifest.data[0].metadata.labels.provider !== 'Red Hat' && (
```

Comment:
Do we know if any operators are still using the old label and check both? We probably need confirmation from the OLM team.

cc @awgreene

Reply:
Hmm I’m unsure how many operators might be using the old label, let me find a way to check and get back to you.

Reply:
we could check for both, just in case. @spadgett WDYT ?

Reply:
I see that quite a lot of operators are still using the label
https://github.com/search?q=org%3Aopenshift%20opsrc-provider&type=code

Reply:
but I also see that some operators like `cluster-manager` has the label set to `Red Hat, Inc`
<img width="1021" alt="Screenshot 2023-07-21 at 17 14 41" src="https://github.com/openshift/console/assets/1668218/49eec519-217f-47ea-9cbf-e4fdee053e2d">

Reply:
I don't think `opsrc-provider` is used anymore:

```
$ oc get packagemanifests -o json | jq -r '.items[] | .metadata.labels["opsrc-provider"]' | sort | uniq -c | sort -n
    525 null
$ oc get packagemanifests -o json | jq -r '.items[] | .metadata.labels["provider"]' | sort | uniq -c | sort -n
      1 Ab Initio
      1 Accuknox
      1 akaris
      1 Alvearie
      1 Anchore Inc.
      1 Ansible
      1 antrea.io
      1 Apicurio
      1 Apicurio Project
      1 APIMatic.io
      1 AppDynamics LLC
      1 Argo CD Community
      1 Backube
      1 boxboat-github-practice
      1 Carlos Eduardo de Paula
      1 catalogic Software
      1 Cisco
      1 Cisco Networking - Compute Solutions
      1 Citrus Framework
      1 cloud-bulldozer
      1 Cockroach Labs
      1 CodeFlare
      1 Commvault
      1 Confluent, Inc.
      1 Craig Trought
      1 CSI Addons Community
      1 CyberArk
      1 DataTrucker.IO
      1 Devfile
      1 devopstales
      1 eamli
      1 Eclipse Foundation
      1 Edgelabs
      1 Elastic
      1 EnterpriseWeb
      1 EPAM Delivery Platform
      1 External Secrets
      1 F5 Networks Inc.
      1 Fossul Framework
      1 Foundation Model Stack
      1 Gingersnap
      1 Grafana Loki SIG Operator
      1 Habana Labs Ltd.
      1 Helm Community
      1 Hitachi
      1 HPE
      1 HPE Ezmeral
      1 Ian Boden
      1 IBM Edge
      1 IBM Quantum
      1 Infinidat
      1 Infinispan
      1 Intel Corporation
      1 JohnSnowLabs
      1 John Snow LABS
      1 Jooho Lee
      1 Juniper Networks
      1 JWS team
      1 KEDA Community
      1 Kepler Operator Contributors
      1 Kong Inc.
      1 Kove
      1 Kubecost
      1 kube-green Community
      1 KubeVirt project
      1 Lenovo (Shanghai) Computer Technology Co., Ltd.
      1 LINBIT
      1 loxilb.io
      1 m88i Labs
      1 Managed Openshift Black Belt
      1 Managed OpenShift Black Belt
      1 Mantisnet
      1 mariadb-operator
      1 MayaData
      1 MemVerge, Inc.
      1 Microcks.io
      1 Microsoft
      1 MuleSoft
      1 NdmSpc project
      1 NEC
      1 NEC Corporation
      1 NetScaler
      1 networkservicemesh.io
      1 NGINX Inc
      1 NooBaa
      1 Nutanix
      1 NVIDIA
      1 NVIDIA Corporation
      1 Odoo Community Association
      1 Omar Gaye IBM
      1 Open Data Hub
      1 OpenShift Database Access
      1 OpenTelemetry Community
      1 Perficient Inc
      1 PingCAP
      1 ProphetStor Data Services, Inc.
      1 Protocol Labs
      1 Provider Name
      1 Pulp Community
      1 QuestLabs
      1 Rebaca Technologies Pvt Ltd
      1 RedHat
      1 Red Hat CCIT
      1 Red Hat Internal
      1 Red Hat Performance
      1 RHsyseng
      1 Ricardo Maraschini
      1 Run:AI Labs
      1 seldon
      1 Silicom.dk
      1 Snyk Ltd.
      1 Solace Corporation
      1 Splunk Inc.
      1 Stakater
      1 Stolostron
      1 Strimzi
      1 Syndesis Team
      1 Thales CPL DIS
      1 The bpfd Community
      1 The cert-manager maintainers
      1 The Flux project
      1 The Shipwright Contributors
      1 The Sigstore Authors
      1 Tigera
      1 Upbound Inc.
      1 Vertica
      1 VictoriaMetrics
      1 VMware
      1 Weaveworks
      1 Windup
      1 Yugabyte Inc
      2 Aerospike
      2 Aqua Security, Inc.
      2 citrix
      2 CloudBees
      2 CNCF
      2 Couchbase
      2 CrowdStrike
      2 DataStax
      2 eG Innovations
      2 Entando, Inc
      2 EnterpriseDB Corporation
      2 Fortanix Inc.
      2 Fujitsu
      2 GitLab Inc
      2 GitLab, Inc.
      2 Instana
      2 Intel® Corporation
      2 Isovalent
      2 JFrog
      2 Kiali
      2 Kubemq.io
      2 MinIO Inc
      2 NetApp
      2 OnGres
      2 Pachyderm, Inc.
      2 Palo Alto Networks
      2 Portworx
      2 Randoli
      2 Red Hat Inc
      2 Red Hat, Inc
      2 Redis
      2 SAP LinuxLab - Red Hat
      2 Seldon Technologies
      2 Silicom Ltd.
      2 SmartX
      2 Solution-Soft
      2 Sonatype
      2 sosivio
      2 Stackable GmbH
      2 Starburst Data, Inc.
      2 Sysdig
      2 The Apache Software Foundation
      2 vFunction
      2 Zabbix LLC
      2 Zettaset
      3 Broadcom, Inc.
      3 Crunchy Data
      3 Datadog
      3 Dynatrace LLC
      3 Hazelcast, Inc
      3 Kasten by Veeam, kasten.io
      3 Konveyor
      3 MongoDB, Inc
      3 NeuVector
      3 opdev
      3 Opstree Solutions
      3 Red Hat Inc.
      3 Trilio
      3 VMware Tanzu
      4 Citrix
      4 Community
      4 Dell Technologies
      4 Intel
      4 Joget, Inc
      4 Veritas Technologies LLC
      5 Keysight
      5 Medik8s
      5 Red Hat Marketplace
      6 Cambridge Semantics Inc.
      7 StreamNative
      9 Percona
      9 Turbonomic, Inc.
     13 Red Hat Community of Practice
     15 IBM
     18 Red Hat, Inc.
     26 Amazon, Inc.
    120 Red Hat
```